pkg/cmd/server/server.go

```diff
@@ -364,3 +361,18 @@ func (c *completedServerConfig) Run(ctx context.Context) error {
 
 	return nil
 }
+
+var promOnce sync.Once
+
+// enableGRPCHistogram protects prom metric registration with a sync.Once.
```

Comment:
I'd say something like: `enableGRPCHistogram` enables the standard time history for gRPC requests, ensuring that it is only enabled once